src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -183,7 +184,45 @@ def get(self, request, organization):
 
         results = list(cursor_result)
 
-        context = serialize(results, request.user, serializer())
+        skip_snuba_fields = {
+            "query",
+            "status",
+            "bookmarked_by",
+            "assigned_to",
+            "unassigned",
+            "subscribed_by",
+            "active_at",
+            "first_release",
+            "first_seen",
+            "message",  # not 100% on this one
```

Comment:
Are you trying to filter out fields that aren't in snuba? `message` is a valid field in snuba.

Reply:
Message is a valid field but it doesn't effect the group event/user counts, it only filters between groups. I ignore it here to determine whether to return a `filtered` field on the group.